templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -1,61 +1,55 @@
 {% extends 'base.html.twig' %}
-
 {% block stylesheets %}
-    {{ encore_entry_link_tags('exception') }}
+    {{ encore_entry_link_tags('web-site') }}
+    <style>
+        .error-code-item {
+            background-color: white;
+            border: 1px solid;
+            margin: 1rem;
+        }
+    </style>
 {% endblock %}
 
-{% block body %}
-
-    <div class="container mt-5">
-    <div class="row">
-        <div class="col-8 offset-2 display-1 text-center">
-
-            <!-- line 01 -->
-            <div class="row text-danger">
-
-                <div class="col-2">
-                    <div class="border border-danger rounded">E</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">O</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">R</div>
-                </div>
-                <div class="col-2">
-                    <div class="border border-danger rounded">!</div>
-                </div>
 
+{% block body %}
+    <div class="container mt-5 has-text-centered" style="width: 40%; margin: 0 auto">
```

Comment:
I guess you can use this in `columns` class like


<div class="columns mt-5 has-text-centered has-text-danger is-size-1"